### ISSUE TITLE
minHeight and minWidth not working

### ISSUE THREAD

**Issue by riccardolorenzon** [DominicTobias/react-image-crop#61]
Hi, 

i set the minHeight and minWidth arguments(in percentage) expecting the component to force a minimum crop area but apparently it does nothing. Any suggestions on how to use that?

**Comment by DominicTobias** [comment]
Hi are you using the latest version? Also post what props you are passing to the component please

**Comment by riccardolorenzon** [comment]
Hi,

I'm using the v1.0.0-rc8. 

The props i pass to the component are: 
`
src={this.props.src}
onImageLoaded={this.onImageLoaded}
onComplete={this.onCropComplete}
crop={crop}
outputPixelUnits={true}
`

where `crop` has these fields: 
`
crop.x = this.props.imageState.x / naturalWidth * 100;
crop.y = this.props.imageState.y / naturalHeight * 100;
crop.height = this.props.imageState.height / naturalHeight * 100;
crop.width = this.props.imageState.width / naturalWidth * 100;

// TODO the following are only test values
crop.minWidth = 25;
crop.minHeight = 25;
`

I would expect to have a min crop area as big as the 25% of the total image area. But apparently it does not work that way.

**Comment by DominicTobias** [comment]
@riccardolorenzon rather than passing the minHeight/maxHeight into the crop object, pass it as a prop - e.g.

```
crop={crop}
minHeight={25}
maxHeight={25}
```

**Comment by DominicTobias** [comment]
Closing - let me know if you have probs with that fix

**Comment by nickensoul** [comment]
Hello!
I've just tried to set `minHeight={100}` and `maxHeight={100}` in [demo codesandbox](https://codesandbox.io/s/72py4jlll6) but it doesn't work, I just cannot select anything at all.
What am I doing wrong?
Thanks!

**Comment by DominicTobias** [comment]
@nickensoul are you trying to pass pixel values? The values are 0-100%. It can be a bit inconvenient but this is how the library is responsive (by using %). You can convert to a % if you like by setting the values in `onImageLoaded` as there you have the image object which has the width and height.

**Comment by nickensoul** [comment]
@DominicTobias thanks for the quick reply!
Yep, according to the [docs](https://github.com/DominicTobias/react-image-crop#minheight-optional) I'm trying to pass number as a percentage values. As mentioned above, I use it this way:
```
<ReactCrop
  src={src}
  crop={crop}
  minHeight={100}
  maxHeight={100}
  onImageLoaded={this.onImageLoaded}
  onComplete={this.onCropComplete}
  onChange={this.onCropChange}
/>
```

But I've got it. If I set e.g. `minHeight={50}`, it works fine, but if more – doesn't. Actually, it works, but unexpected: if `minHeight` is 80, then start position of drag must be such that actual selection height' height is at least 80. In other worlds, start position of the drug may be in edges of image only at Y axis, e.g. for `minHeight={80}`, it is 20% of image on its edges. So if `minHeight={100}`, it's impossible to get position of start at <= 0% or >=100% of image height in coordinates.

Sorry for my English, that's was hard to explain 😆
